MINA_COMMIT

```diff
@@ -1,2 +1,2 @@
 The mina commit used to generate the backends for node and chrome is
-532588af567b7e9a6c157163a8bacc3b2491160f
+2f263afe409c69b19a60b8f7235cf3f0892e33ec
```

Comment:
I guess this is what generated `snarky_js_chrome.bc.js`? It'd be nice if CI verifies that the file hasn't been manually modified as well :P

Reply:
Hm yeah, it would already be a win if my build script updated this file automatically, right now I do it manually